solr/packaging/build.gradle

```diff
@@ -250,11 +251,18 @@ task integrationTests(type: BatsTask) {
 
     // TODO - if quiet then don't tee
     standardOutput = new TeeOutputStream(System.out, new FileOutputStream("$integrationTestOutput/test-output.txt"))
-
+    println("Runnin BATS tests with Solr base port ${solrPort}")
   }
 
   environment SOLR_TIP: distDir.toString()
   environment SOLR_HOME: solrHome
+  environment SOLR_PID_DIR: solrHome
+  environment SOLR_PORT: solrPort
+  environment SOLR2_PORT: solrPort + 1
+  environment SOLR3_PORT: solrPort + 2
+  environment SOLR_PORT_8983: 8983
+  environment SOLR_PORT_7574: 7574
```

Comment:
not sure we need solr3 or solr_port8983 and solr_port7574....

Reply:
Removed. There was a test somewhere that used a third port number (check the var), but perhaps it could use 1 or 2?

Reply:
We do need a Solr 3. The new mTLS tests spin up a third node. They could restart the second node instead, so your choice there.

Reply:
Keeping as is for now.